pinot-common/src/main/java/com/linkedin/pinot/common/exception/QueryException.java

```diff
@@ -60,6 +62,8 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTrace) {
   public static final ProcessingException COMBINE_SEGMENT_PLAN_TIMEOUT_ERROR =
       new ProcessingException(COMBINE_SEGMENT_PLAN_TIMEOUT_ERROR_CODE);
   public static final ProcessingException QUERY_EXECUTION_ERROR = new ProcessingException(QUERY_EXECUTION_ERROR_CODE);
+  public static final ProcessingException SERVER_SCHEDULER_DOWN_ERROR = new ProcessingException(SERVER_SHUTTING_DOWN_ERROR_CODE);
```

Comment:
Can you please add a TODO to handle these errors in broker appropriately? Or, do you plan to make that checkin soon?